packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Inputs/index.tsx

```diff
@@ -0,0 +1,281 @@
+                            ? quotesRequest.fiatStringAmount
+                            : ''
+                    }
+                    innerRef={register({
+                        validate: (value: string) => {
+                            if (activeInput === fiatInput) {
+                                if (!value) {
+                                    if (formState.isSubmitting) {
+                                        return <Translation id="TR_BUY_VALIDATION_ERROR_EMPTY" />;
+                                    }
+                                    return;
+                                }
+
+                                if (!isDecimalsValid(value, 2)) {
+                                    return <Translation id="TR_BUY_VALIDATION_ERROR_NOT_NUMBER" />;
+                                }
+
+                                if (amountLimits) {
+                                    const amount = Number(value);
+                                    if (amountLimits.minFiat && amount < amountLimits.minFiat) {
+                                        return (
+                                            <Translation
+                                                id="TR_BUY_VALIDATION_ERROR_MINIMUM_FIAT"
+                                                values={{
+                                                    minimum: amountLimits.minFiat,
+                                                    currency: amountLimits.currency,
+                                                }}
+                                            />
+                                        );
+                                    }
+                                    if (amountLimits.maxFiat && amount > amountLimits.maxFiat) {
+                                        return (
+                                            <Translation
+                                                id="TR_BUY_VALIDATION_ERROR_MAXIMUM_FIAT"
+                                                values={{
+                                                    maximum: amountLimits.maxFiat,
+                                                    currency: amountLimits.currency,
+                                                }}
+                                            />
+                                        );
+                                    }
+                                }
+                            }
+                        },
+                    })}
+                    onFocus={() => {
+                        setActiveInput(fiatInput);
+                    }}
+                    onChange={() => {
+                        setValue(cryptoInput, '');
+                        clearErrors(cryptoInput);
+                    }}
+                    state={errors[fiatInput] ? 'error' : undefined}
+                    name={fiatInput}
+                    // @ts-ignore TODO
+                    bottomText={errors[fiatInput] && errors[fiatInput].message}
```

Comment:
use `@wallet-components/InputError` like in send inputs

`<InputError error={errors[fiatInput]} />`